v1.1/insert.md

```diff
@@ -44,6 +44,15 @@ Parameter | Description
 
 ## Examples
 
+First let's create a table accounts.
+
+~~~ sql
+> CREATE TABLE accounts(
+    id INT NOT NULL UNIQUE,
```

Comment:
The `SHOW` output below shows the `id` column using `unique_rowid()` as the default value. Can you add that to this `CREATE TABLE`?